src/System.Binary.Base64/System/Binary/Base64.cs

```diff
@@ -1,30 +1,39 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Text.Utf8;
 using System.Runtime.CompilerServices;
 
-namespace System.Binary
+namespace System.Binary.Base64
 {
     public static class Base64
     {
         static string s_characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
-        static byte[] s_encodingMap = new Utf8String(s_characters).CopyBytes();
-        static byte[] s_decodingMap = new byte[123];
+        static readonly byte[] s_encodingMap = GetEncodingMap(s_characters);
+        static readonly byte[] s_decodingMap = GetDecodingMap(s_characters);
+        static readonly byte s_encodingPad = s_encodingMap[64];
 
-        static Base64()
+
+        private static byte[] GetEncodingMap(string str)
+        {
+            var data = new byte[str.Length];
+            Text.TextEncoder.Utf8.TryEncode(str, data, out int written);
```

Comment:
Isn't this just casting each char to a byte? This would be pretty trivial to turn into constant data.